### PR TITLE
Update 0.48mm Standard @BBL X1C 0.8 nozzle.json

### PR DESCRIPTION
**Pull request by SufficentMuffin**
Removed P1S 0.6mm nozzle from process profile.

### CODE REVIEW

**Comment by SoftFever** [comment]
Hi,

Can you provide some description/explain for the change?

**Comment by SufficentMuffin** [comment]
Of course, please excuse.

If you choose Bambu´s P1S with the 0.6 mm nozzle the Slicer will show the 0.48 mm layer height profile for it. 
This should not be the case since the max layer height for 0.6 mm nozzle is 0.42 mm. 
If you choose the 0.48mm profile it will throw an "error" whichs says the layer height was changed back to 0.42mm.

**Comment by SoftFever** [comment]
> Of course, please excuse.
> 
> If you choose Bambu´s P1S with the 0.6 mm nozzle the Slicer will show the 0.48 mm layer height profile for it. This should not be the case since the max layer height for 0.6 mm nozzle is 0.42 mm. If you choose the 0.48mm profile and slice an object it will throw an "error" whichs says the layer height was changed back to 0.42mm.

Thanks for the info and nice catch.
I think in this case, it would be better to change it to `Bambu Lab P1S 0.8 nozzle`.

**Comment by SufficentMuffin** [comment]
Sorry if I confused you.

I don't want to print with 0.48 mm layer height.

I just wanted to point out that it's not correct that you can choose 0.48mm while 0.6mm nozzle is active. That's why I removed the line from the code.

**Comment by SufficentMuffin** [comment]
I can provide some screenshots later to make it more clear :)

**Comment by SoftFever** [comment]
> Sorry if I confused you.
> 
> I don't want to print with 0.48 mm layer height.
> 
> I just wanted to point out that it's not correct that you can choose 0.48mm while 0.6mm nozzle is active. That's why I removed the line from the code.

Let me clarify what I meant.
The 0.48mm process profile is meant for 0.8mm nozzle.
P1S 0.6 nozzle machines shouldn't see this process. Which your PR is aiming to fix.
What I suggested change it to P1S 0.8 nozzle as it is intended for.
image

File: resources/profiles/BBL/process/0.48mm Standard @BBL X1C 0.8 nozzle.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "compatible_printers": [
         "Bambu Lab X1 Carbon 0.8 nozzle",
         "Bambu Lab X1 0.8 nozzle",
-        "Bambu Lab P1S 0.6 nozzle",
+        "Bambu Lab P1S 0.8 nozzle",
```
Comment (SoftFever) [inline resources/profiles/BBL/process/0.48mm Standard @BBL X1C 0.8 nozzle.json:11]: @SufficentMuffin 
I pushed a change here.
I think this is the better communication way :)
Now you won't see 0.48 for P1S 0.6 nozzle.
And for P1S 0.8 nozzle, it's available.

**Review by SoftFever — APPROVED** [review]

**Pull request merged**